### PR TITLE
Add dockbit.com badge

### PR DESCRIPTION
**Pull request by assimovt**
This adds https://dockbit.com deployment status badge for a given Dockbit organization pipeline. I'd really appreciate your review and merge. 🚀 Thx!

### CODE REVIEW

File: logo/dockbit.svg [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (Daniel15) [inline logo/dockbit.svg]: Please remove the empty `<g>` elements. They don't do anything when they're empty  :)

File: logo/dockbit.svg [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (Daniel15) [inline logo/dockbit.svg]: Please remove this comment

File: logo/dockbit.svg [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (Daniel15) [inline logo/dockbit.svg]: Remove unneeded ID `id="Layer_1"`

File: server.js [hunk 4]
```diff
@@ -4789,6 +4789,53 @@ cache(function(data, match, sendBadge, request) {
...
 
+camp.route(/^\/dockbit\/([A-Za-z0-9-_]+)\/([A-Za-z0-9-_]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var org      = match[1];
+  var pipeline = match[2];
+  var format   = match[3];
+
+  var token     = data.token;
+  var badgeData = getBadgeData('deploy', data);
+  var apiUrl    = 'https://dockbit.com/' + org + '/' + pipeline + '/status/' + token;
```
Comment (Daniel15) [inline server.js]: You can use a template string to make this a bit nicer:
```
const apiUrl = `https://dockbit.com/${org}/${pipeline}/status/${token}`
```

**Comment by assimovt** [comment]
@Daniel15 Thanks for the review. I've added the changes you requested and tests as well. Ready to :shipit:

**Comment by Daniel15** [comment]
Thank you! :shipit:

**Comment by assimovt** [comment]
@Daniel15 Thanks for the merge. Any ideas when it's gonna be publicly available at https://shields.io/ ?

**Comment by assimovt** [comment]
@Daniel15 Any chance to get it deployed? Or what's the process like to get it shipped? 😊

**Comment by Daniel15** [comment]
Sorry for the delay! @espadrine handles the deployment side of things, so it's up to him. 👍

**Comment by assimovt** [comment]
@espadrine would u mind answering how often is the site deployed?

**Comment by assimovt** [comment]
@espadrine @Daniel15 Is there anyone else who can push the update, please?

**Pull request merged**